### PR TITLE
HCAP-1047 | HCAP-1030 | Create Service for Post Hire Flow and Update existing services.

### PR DESCRIPTION
**Pull request by fwpushan**
- Create service module with create and getter 
- Update existing `getParticipants` to include post-hire-statuses in response 
- Add and update tests.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -32,6 +32,7 @@ yarn-error.log*
 .vscode
 .eslintcache
 key
+coverage
```
Comment (dbayly-freshworks) [inline .gitignore:35]: What is in the coverage folder? Do I need up update my local?

Comment (fwpushan) [reply .gitignore:35]: Test coverage report folder. This will address the issue.

File: server/tests/participant.service.test.js [hunk 2]
```diff
@@ -1023,4 +1026,44 @@ describe('Participants Service', () => {
...
+
+    const participants = await getParticipants(
+      { isEmployer: true, id: employerAId, regions },
+      null,
+      null,
+      null,
+      null,
+      null,
+      null,
+      null,
```
Comment (dbayly-freshworks) [inline server/tests/participant.service.test.js:1064]: This feels like a bit of a code smell here. We may want to refactor the parameters of the getParticipants function.

Comment (fwpushan) [reply server/tests/participant.service.test.js:1064]: we may need to refactor getParticipants func. But this is not in the scope of this sprint.

File: server/services/participants.js [hunk 3]
```diff
@@ -420,6 +421,17 @@ const getParticipants = async (
     total: Number(await table.count(criteria || {})),
   };
 
+  // HCAP:1030: Get participants post-hire statuses
+  participants = await Promise.all(
```
Comment (dbayly-freshworks) [inline server/services/participants.js:425]: What are your thoughts around adding post hire statuses to the response  body of the main getParticipants call?

Comment (fwpushan) [reply server/services/participants.js:425]: in participant view we have to show graduation status ....

**Review by dbayly-freshworks — APPROVED** [review]

**Pull request merged**